debian/rules

```diff
@@ -95,8 +95,8 @@ override_dh_install:
 		"debian/tmp/usr/share/host-jdks/jdk/windows_x86/jdk1.8/jdk.zip"
 
 	./scripts/fetch-file-from-artifactory.sh \
-		"windows/jdk1.8/OpenJDK8U-jdk_x64_windows_hotspot_8u402b06-manifest" \
-		"0726e6616a2d6c27a7c86cba997f55cc0c9bd92af5722497adb4afedf6acbeff" \
+		"windows/jdk1.8/OpenJDK8U-jdk_x64_windows_hotspot_8u402b06-sha256manifest" \
```

Comment:
Yeah, that's a good idea. However, since we are updating the rules files, it's difficult to write tests for them. But we can update the documentation that we are following to update this version. Let me do that.

Reply:
Added Note here: https://delphix.atlassian.net/wiki/spaces/SL/pages/12985598939/Upgrade+Java+Minor+version+-+Linux